UPGRADE.md

```diff
@@ -16,12 +16,16 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+### [shopsys/framework]
 - for [product search via Elasticsearch](/docs/introduction/product-search-via-elasticsearch.md), you'll have to:
     - check changes in the `docker-compose.yml` template you used and replicate them, there is a new container with Elasticsearch
     - run `docker-compose up -d`
     - update composer dependencies `composer update`
     - create Elasticsearch indexes by running `php phing elasticsearch-indexes-create`
     - export products into Elasticsearch by `php phing elasticsearch-products-export`
+- check for your usages of advert position names constants, they were removed from `Advert` class

```

Comment:
Maybe we should just leave the constants in there (maybe marked as `@deprecated`?) so we won't make an unnecessary BC-break...

Reply:
You haven't even removed them! :D